InstabugSample/__tests__/instabugUtils.spec.js

```diff
@@ -56,25 +56,25 @@ describe('Test global error handler', () => {
 
     });
 
-    it('should emit event IBGSendUnhandledJSCrash when platform is android and onReportSubmitHandler is set', (done) => {
```

Comment:
Why are we removing this?

Reply:
This is the same test we commented in a diffferent PR, the one that broke even on previous passing builds. Seems to be a problem with the test